### PR TITLE
WIP Feat/moved mfa to org level

### PR DESCRIPTION
**Pull request by sheensantoscapadngan**
# Description 📣
WIP

## Type ✨

- [ ] Bug fix
- [ ] New feature
- [ ] Improvement
- [ ] Breaking change
- [ ] Documentation

# Tests 🛠️

```sh
# Here's some code block to paste some code snippets
```

---

- [ ] I have read the [contributing guide](https://infisical.com/docs/contributing/getting-started/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/getting-started/code-of-conduct). 📝

### CODE REVIEW

**Comment by sheensantoscapadngan** [comment]
The failing CLI tests are due to an old PR that updated the error codes being thrown by our backend service... we forgot to update the test snapshot files accordingly.

**Review by maidul98 — COMMENTED** [review]
Almost there but missing some small edge cases. I have sent them to you via slack

File: backend/src/server/routes/v1/sso-router.ts [hunk 1]
```diff
@@ -280,10 +280,6 @@ export const registerSsoRouter = async (server: FastifyZodProvider) => {
         providerAuthToken: req.body.providerAuthToken
       });
```
[the remark this answers is not in the document]
Comment (sheensantoscapadngan) [reply backend/src/server/routes/v1/sso-router.ts:281]: this is intentional. I didn't want to break existing clients that checked for `mfaEnabled` so I left it as it is

File: backend/src/server/routes/v3/login-router.ts [hunk 2]
```diff
@@ -60,14 +61,21 @@ export const registerLoginRouter = async (server: FastifyZodProvider) => {
...
+
       void res.setCookie("jid", tokens.refresh, {
         httpOnly: true,
         path: "/",
         sameSite: "strict",
         secure: cfg.HTTPS_ENABLED
       });
 
-      return { token: tokens.access };
+      return { token: tokens.access, isMfaEnabled: false };
```
[the remark this answers is not in the document]
Comment (sheensantoscapadngan) [reply backend/src/server/routes/v3/login-router.ts:78]: MFA happens after select organization now

**Review by maidul98 — APPROVED** [review]
